src/Uno.Toolkit.RuntimeTests/Tests/ResponsiveExtensionsTests.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Uno.UI.RuntimeTests;
+using Uno.Toolkit.RuntimeTests.Helpers;
+using Uno.Toolkit.UI.Helpers;
+using Windows.Foundation;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI;
+using Microsoft.UI.Xaml.Media;
+#else
+using Windows.UI.Xaml.Controls;
+using Windows.UI;
+using Windows.UI.Xaml.Media;
+#endif
+
+namespace Uno.Toolkit.RuntimeTests.Tests;
+
+[TestClass]
+[RunsOnUIThread]
+internal class ResponsiveExtensionsTests
```

Comment:
just move the `#if !IS_UWP || HAS_UNO` conditional to wrap the class, instead of each methods